.travis.yml

```diff
@@ -7,6 +7,8 @@ go:
 
 env:
   - MIGRATE_TEST_CONTAINER_BOOT_TIMEOUT=60
+  - secure: "ENCYRYPTED USERNAME"
```

Comment:
Remove environment variables for now since it's creating extra TravisCI builds. I'll figure out a way to safely pass in credentials once this PR is merged.

Reply:
To add encrypted variable you should do this with `DOCKER_USERNAME` and `DOCKER_PASSWORD` (cf https://docs.travis-ci.com/user/environment-variables/#Defining-encrypted-variables-in-.travis.yml)

```
$ travis encrypt DOCKER_USERNAME=username --add env.global
$ travis encrypt DOCKER_PASSWORD=password --add env.global
```